climakitae/ui/select.py

```diff
@@ -0,0 +1,212 @@
+import panel as pn
+from shapely.geometry import box, Polygon
+from matplotlib.figure import Figure
+import matplotlib.ticker as ticker
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+
+from climakitae.core.data_interface import DataParametersWithPanes
+
+
+class Select:
+    def __init__(self):
+        self.data_parameters = DataParametersWithPanes()
```

Comment:
I might call this object 'selections' instead of 'data_parameters' for greater consistency with the old version.

Reply:
Why can't Select inherit from DataParametersWithPanes instead actually?

Reply:
It can! Like I said I initially ran into some conceptual issues but looks like now it works well! I will make the changes for this.

Reply:
> I might call this object 'selections' instead of 'data_parameters' for greater consistency with the old version.

I will do this

Reply:
> I might call this object 'selections' instead of 'data_parameters' for greater consistency with the old version.

This will go away with subclassing the GUI classes to the Parameter classes.